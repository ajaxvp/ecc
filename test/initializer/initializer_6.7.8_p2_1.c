/* ISO: 6.7.8 (2); out of bounds initializations */

int ys[1] = { 5, 6 };

int main(void)
{
    int xs[1] = { 2, 3 };

    (int[1]){ 3, 5 };
}
