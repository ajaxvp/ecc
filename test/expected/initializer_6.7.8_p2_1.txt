ecc: error: [3:18] this initializer and any after it would write outside the object being initialized
ecc: error: [7:22] this initializer and any after it would write outside the object being initialized
ecc: error: [9:18] this initializer and any after it would write outside the object being initialized
